src/gui/gui.cc

```diff
@@ -271,6 +272,19 @@ void PCSX::GUI::startFrame() {
     glfwPollEvents();
     SDL_PumpEvents();
```

Comment:
You definitely want to get rid of this `SDL_PumpEvents()` call here, otherwise events will get eaten in it, and the `SDL_PollEvent(&event)` right after will not see them.

```suggestion
```